bodhi/server/templates/master.html

```diff
@@ -24,8 +24,10 @@
     % else:
         <title>${update.title} - Fedora Updates System</title>
     % endif
-    <link href="${request.static_url('bodhi:server/static/bootstrap/css/bootstrap.css') }" rel="stylesheet" />
-    <link href="${request.static_url('bodhi:server/static/font-awesome-4.4.0/css/font-awesome.min.css') }" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.1/fedora-bootstrap.css" rel="stylesheet" />
+    <link rel='stylesheet' href='//cdn.jsdelivr.net/font-hack/2.020/css/hack.min.css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,700,700italic' rel='stylesheet' type='text/css'>
+    <link href="https://ryanlerch.fedorapeople.org/font-awesome.css" rel="stylesheet" />
```

Comment:
For the other apps, we have been using the resources on apps.fp.org/global for these resources. I have updated these links to use these resources now.

One note, however, once i make this change to use the fontawesome from apps.fp.o, the fontawesome icons won't show up anymore when testing locally. This is because apps.fp.o has CORS restriuctions on non-fedora domains. I have [submitted a patch](https://lists.fedoraproject.org/archives/list/infrastructure@lists.fedoraproject.org/thread/F7JPDKZS25NJPHTWXXLYN6GNUJTBYU44/) to add the local fallback as a last resort in font-awesome.css, so it will work when testing locally if you install the font awesome fonts locally from the fedora repos.